Assets/Scripts/PlayerManager.cs

```diff
@@ -35,6 +36,12 @@ private void Start()
 
     private void Update()
     {
+        // Context switch was requested
```

Comment:
Add a comment to ensure this code is always before any interactions with a controller (to prevent issues with using a different controller midway through this method)